src/Files.App/ViewModels/UserControls/Widgets/FileTagsWidgetViewModel.cs

```diff
@@ -73,6 +73,9 @@ public override List<ContextMenuFlyoutItemViewModel> GetItemMenuItems(WidgetCard
 		{
 			return new List<ContextMenuFlyoutItemViewModel>()
 			{
+				new ContextMenuFlyoutItemViewModelBuilder(CommandManager.OpenInNewTabFromHomeAction).Build(),
+				new ContextMenuFlyoutItemViewModelBuilder(CommandManager.OpenInNewWindowFromHomeAction).Build(),
+				new ContextMenuFlyoutItemViewModelBuilder(CommandManager.OpenInNewPaneFromHomeAction).Build(),
```

Comment:
Are these supposed to be here?

Reply:
In general, any other context menus have them at the top. Should they be placed as were?

Reply:
@0x5bfa

Reply:
Looks like the same 🤔
Maybe because Open With didn't work properly in stable.

Before|After
---|---
![image](https://github.com/files-community/Files/assets/62196528/ee457205-90ec-474d-bde0-b54ffaee40f5)|![image](https://github.com/files-community/Files/assets/62196528/b151dd4a-5be5-4b5a-8ade-135d815688da)

Reply:
Are you right clicking a folder or file?

Reply:
"Open in new tab" is for folders whereas "Open with" is for files.

Reply:
Oh

Reply:
Could you check in your stable install?

Reply:
Which widget are you right clicking?

Reply:
![image](https://github.com/files-community/Files/assets/39923744/68f426fb-0fbb-4ae7-9d73-5020a3b82134)

Reply:
Damn. I'll fix.